corehq/apps/app_manager/views/download.py

```diff
@@ -39,6 +40,8 @@ def _get_profile(request):
     else:
         return None
 
+
+@require_deploy_apps
```

Comment:
Isn't this the file that apps download to get the app before logging in? If that's the case we can't assume that they have a user